### PR TITLE
updpatch: rust 1:1.71.1-1

### PR DESCRIPTION
**Pull request by hack3ric**
Fix rotten patch.

### CODE REVIEW

File: rust/riscv64.patch [hunk 1]
```diff
@@ -103,26 +101,3 @@ index 99f0b93..56bbbc7 100644
...
--  provides=(lib32-rust)
--  conflicts=(lib32-rust)
--  replaces=(lib32-rust)
--
--  cp -a dest-i686/* "$pkgdir"
--
--  mkdir -p "$pkgdir/usr/share/licenses"
--  ln -s rust "$pkgdir/usr/share/licenses/$pkgname"
--}
--
```
Comment (r-value) [inline rust/riscv64.patch:125]: why are these removals removed?

Comment (hack3ric) [reply rust/riscv64.patch:125]: The `package_$name` functions never gets called if `$pkgname` does not contain `$name`.

Comment (r-value) [reply rust/riscv64.patch:125]: I think we'd better keep the `PKGBUILD` clean

Comment (hack3ric) [reply rust/riscv64.patch]: Added back.

Comment (Xeonacid) [reply rust/riscv64.patch]: > I think we'd better keep the `PKGBUILD` clean

I think it would be better to keep a minimal patch for less rotten patch.

**Pull request merged**